### ISSUE TITLE
Latest download file does not include scoped css files

### ISSUE THREAD

**Issue by missymar** [salesforce-ux/design-system#231]
The only files found in the new downloaded `styles` folder are:

salesforce-lightning-design-system.css 
salesforce-lightning-design-system.min.css

Have the other scoped versions been removed intentionally? I downloaded 2.0.1 via the Downloads page.

**Comment by stefsullrew** [comment]
Hi @missymar - Yes, they were removed intentionally. If you're serving your component within the Lightning Experience (on the platform), the CSS is now included automatically. There is no longer a need to scope your document.

If you are working on a Lightning App (app.css) or Visualforce or in Lightning Out, you will still, at least for now, need to scope your file. But we are now recommending that for those, you use a custom scope (to something unique, like your company name). We've provided a tool. There are explanations in the readme file as well as on the downloads page. But here's a link to it here: https://tools.lightningdesignsystem.com/css-customizer
